app/src/main/res/values/strings.xml

```diff
@@ -5,8 +5,8 @@
     <string name="action_search">Search</string>
     <string name="action_add">Add</string>
 
-    <string name="noGiftCards">Click the "+" (plus) button to add a card first.\n\nCatima carries your cards on the device, so they are always within reach.</string>
-    <string name="noMatchingGiftCards">Didn\'t find anything. Try changing your search.</string>
+    <string name="noGiftCards">Click the "+" (plus) button to add a card first.</string>
```

Comment:
The main screen is already very empty on first launch. I'd rather see the word "first" removed and maybe a reference added to the import menu setting.

Something like

"""
Catima carries your cards on the device, so they are always within reach.

To get started, add your first card by pressing the "+" (plus) button or import your data from another app by selecting Import/Export in the top-right dropdown menu.
"""

Reply:
I don't want to get started, I want to start.
~Give me the bare minimum, not a thing to read.

Reply:
Still, now we're changing the string anyway, it's probably a good moment to add something about the support for importing from some other apps.